src/index.tsx

```diff
@@ -116,7 +122,9 @@ wrapComponents({
               },
             }}
           >
-            <DefaultComponent {...props} />
+            <SnackbarProvider styleProps={{}}>
```

Comment:
@bobular thank you for your review and tests. Actually I discussed with @jtlong3rd about this and he recommended to place the SnackbarProvider in there: iirc we first thought of WorkspaceRouter but later changed our mind.

Reply:
For qa/production, I would place the `SnackbarProvider` here (similar to what Ann did for `MicrobiomeWebsite`):

https://github.com/VEuPathDB/ClinEpiWebsite/blob/3da5089fe042261288ddac3a51577a686053af7d/Site/webapp/js/client/component-wrappers/Page.tsx#L24-L25

This will configure Snackbar for all client code in `ClinEpiWebsite` (not just EDA).

Reply:
@jtlong3rd so it looks like additional work is required. I don't have ClinEpiWebsite env so can you please do that on my behalf? 👍